sbin/build.sh

```diff
@@ -804,6 +804,8 @@ generateSBoM() {
   addSBOMComponentPropertyFromFile "${javaHome}" "${classpath}" "${sbomJson}" "Eclipse Temurin" "make_command_args" "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/makeCommandArg.txt"
 
   # Below add build tools into metadata tools
+  # Add GLIBC version
+  addSBOMMetadataTools "${javaHome}" "${classpath}" "${sbomJson}" "GLIBC" "rpm -q glibc"
```

Comment:
This will only work on `rpm`-based distributions. For arm32 and RISC-V we build on Ubuntu so `dpkg` will need to be used to identify the libc version used for the build.

(Also noting that  if we ever start using sysroots to run the build then this will not be the correct way to detect what was used, but that can be addressed if and when we make such a change)

Reply:
@sxa yeah, you're right, and your point on sysroot is also key, given that can have been specified as a build arg...
I think the following will work based on using the compiler, but a little fiddly to work out:
1. Get $CC and $SYSROOT_CFLAGS from the build's generated spec.gmk: "build/*/spec.gmk" file
2. Call:

Reply:
@chandrashritii The spec.gmk file in build.sh can be located from variables:
```
grep "^CC :=" ${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/build/*/spec.gmk
grep grep "^SYSROOT_CFLAGS :=" ${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/build/*/spec.gmk
```

Reply:
Similarly we can also get the Gcc version easily from that:
```
"^CC_VERSION_NUMBER :="
```